extensions/wikia/VideoHandlers/js/handlers/Ooyala.js

```diff
@@ -72,6 +72,10 @@ define('wikia.videohandler.ooyala', [
 				}
 			});
 
+			if (typeof params.onCreate === 'function') {
+				params.onCreate(player);
+			}
+
 			// Log all events and values (for debugging)
 			/*messageBus.subscribe('*', 'tracking', function(eventName, payload) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ACommentedCode)